#! /bin/sh /usr/share/dpatch/dpatch-run
## 82_ssl.dpatch by Mike Hommey <glandium@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Disable SSLv2 and weak ciphers.

@DPATCH@

=== netwerk/base/public/security-prefs.js
==================================================================
--- xulrunner/netwerk/base/public/security-prefs.js	(revision 78)
+++ xulrunner/netwerk/base/public/security-prefs.js	(local)
@@ -1,6 +1,6 @@
 pref("general.useragent.security",       "U");
 
-pref("security.enable_ssl2",             true);
+pref("security.enable_ssl2",             false);
 pref("security.enable_ssl3",             true);
 pref("security.enable_tls",		 true);
 
@@ -18,8 +18,8 @@
 pref("security.ssl3.rsa_des_sha", true);
 pref("security.ssl3.rsa_1024_rc4_56_sha", true);
 pref("security.ssl3.rsa_1024_des_cbc_sha", true);
-pref("security.ssl3.rsa_rc4_40_md5", true);
-pref("security.ssl3.rsa_rc2_40_md5", true);
+pref("security.ssl3.rsa_rc4_40_md5", false);
+pref("security.ssl3.rsa_rc2_40_md5", false);
 pref("security.ssl3.dhe_rsa_aes_256_sha", true);
 pref("security.ssl3.dhe_dss_aes_256_sha", true);
 pref("security.ssl3.rsa_aes_256_sha", true);
